tests/preprocessing/test_preprocessor.py

```diff
@@ -178,3 +178,64 @@ def test_mutable_train_data_fit_transform(self, mocker: MockerFixture):
             )
         assert "new_column" not in train_data.columns
         assert "new_column" in result.columns
+    
+
+
+    @pytest.mark.parametrize(("input, expected"), 
+    [
+        # example 1
+        (pd.DataFrame({
+            "a":[1,8,np.nan],

```

Comment:
Don't forget spaces after a comma
```suggestion
            "a":[1, 8 , np.nan],

```